src/compiler/checker.ts

```diff
@@ -11458,8 +11471,25 @@ namespace ts {
                 const containingClassSymbol = getSymbolOfNode(containingClassDecl);
                 const containingClassInstanceType = <InterfaceType>getDeclaredTypeOfSymbol(containingClassSymbol);
                 const baseConstructorType = getBaseConstructorTypeOfClass(containingClassInstanceType);
+                const statements = (<Block>node.body).statements;
+                let superCallStatement: ExpressionStatement;
+                let isSuperCallFirstStatment: boolean;
 
-                if (containsSuperCall(node.body)) {
+                for (const statement of statements) {
+                    if (statement.kind === SyntaxKind.ExpressionStatement && isSuperCallExpression((<ExpressionStatement>statement).expression)) {
+                        superCallStatement = <ExpressionStatement>statement;
+                        if (isSuperCallFirstStatment === undefined) {
+                            isSuperCallFirstStatment = true;

```

Comment:
I didn't break because i want to get `superCallStatement` to report an error on line  [11513] (https://github.com/Microsoft/TypeScript/pull/5833#discussion-diff-50729856R11513) though that is no longer true so I will clean it up